includes/admin/settings/CarrierSettings.php

```diff
@@ -248,6 +249,22 @@ private function createDefaultExportSettingsArray(string $option): array
                     'type'      => 'toggle',
                 ];
                 break;
+            case AbstractConsignment::SHIPMENT_OPTION_HIDE_SENDER:
+                $settings[] = [
+                    'name'      => WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_HIDE_SENDER,
+                    'label'     => __('shipment_options_expedition_secret', 'woocommerce-myparcel'),
```

Comment:
de taal string graag aanpassen naar shipment_options_hide_sender, dat had ik volgens mij ook gedaan? En zo staat het nu ook in de vertalingen sheet volgens mij? (ik zie nu dat ik het alleen verderop had aangepast... my bad)